internal/users/users.go

```diff
@@ -0,0 +1 @@
+package users
```

Comment:
Is there a specific reason why we have this empty file over here?

Reply:
> Each folder in golang apparently needs to have a file with the same name
This is definitely a good heuristic to follow, however it's not something that's enforced anywhere. IMO we could remove the `users.go` file and leave the `users_embed.go` untouched.

Reply:
For some reason, I thought removing `user.go` broke the compile/build so I left it, but I just deleted it, and it doesn't look like it so I'll just leave `users_embed.go`.